IBAnimatable/PresentExplodeSegue.swift

```diff
@@ -10,4 +11,8 @@ public class PresentExplodeSegue: UIStoryboardSegue {
     destinationViewController.transitioningDelegate = PresenterManager.sharedManager().retrievePresenter(.Explode(params: []))
     sourceViewController.presentViewController(destinationViewController, animated: true, completion: nil)
   }
+  public func perform(completion: OptionalAnimatableCompletion) {
```

Comment:
We could do it in `AnimatableViewController` as a `performSegueWithIdentifier` method.